ra2ce/analysis/losses/multi_link_redundancy.py

```diff
@@ -58,7 +60,7 @@ def _update_time(
         ):
             return df_calculated, gdf_graph
```

Comment:
You could bring this `if-else` statement as the first (fast exit goes first).

Reply:
Adjusted